src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue30690.cs

```diff
@@ -0,0 +1,111 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue30690 : _IssuesUITest
```

Comment:
Add a test that actually performs a pull to refresh and checkl the status of that interaction.